src/platform/intel/cavs/platform.c

```diff
@@ -146,13 +146,8 @@ EXT_MAN_PORT(
 #endif
 
 #if CONFIG_CANNONLAKE || CONFIG_ICELAKE || CONFIG_TIGERLAKE
-#if CONFIG_CAVS_LPRO
-#define CAVS_DEFAULT_RO		SHIM_CLKCTL_RLROSCC
-#define CAVS_DEFAULT_RO_FOR_MEM	SHIM_CLKCTL_OCS_LP_RING
-#else
-#define CAVS_DEFAULT_RO		SHIM_CLKCTL_RHROSCC
-#define CAVS_DEFAULT_RO_FOR_MEM	SHIM_CLKCTL_OCS_HP_RING
-#endif
+#define CAVS_ROS	(SHIM_CLKCTL_RHROSCC | SHIM_CLKCTL_RLROSCC)
```

Comment:
Why is CONFIG_CAVS_LPRO support removed here? 
You could request LPRO in #if and request both in #else to get the same result with CAVS_LPRO removed from the TGL config and still support LPRO-only configuration.

Reply:
I can't imagine the usage of the scale down scenario, to avoid confusion, I just removed it.

Reply:
@keyonjie it looks like we really only have 2 options right? #if CONFIG_CAVS_LPRO is enabled, we do not enable HPRO but LPRO is enabled whether this config is selected or not. So, we should keep the #if CONFIG_CAVS_LPRO support here